sktime/forecasting/var.py

```diff
@@ -184,6 +185,112 @@ def _predict(self, fh, X=None):
         )
         return y_pred
 
+    def _predict_interval(self, fh, X=None, coverage: [float] = None):
+        """Compute/return prediction quantiles for a forecast.
+
+        private _predict_interval containing the core logic,
+            called from predict_interval and possibly predict_quantiles
+
+        State required:
+            Requires state to be "fitted".
+
+        Accesses in self:
+            Fitted model attributes ending in "_"
+            self.cutoff
+
+        Parameters
+        ----------
+        fh : guaranteed to be ForecastingHorizon
+            The forecasting horizon with the steps ahead to to predict.
+        X : optional (default=None)
+            guaranteed to be of a type in self.get_tag("X_inner_mtype")
+            Exogeneous time series for the forecast
+        coverage : list of float (guaranteed not None and floats in [0,1] interval)
+           nominal coverage(s) of predictive interval(s)
+
+        Returns
+        -------
+        pred_int : pd.DataFrame
+            Column has multi-index: first level is variable name from y in fit,
+                second level coverage fractions for which intervals were computed.
+                    in the same order as in input `coverage`.
+                Third level is string "lower" or "upper", for lower/upper interval end.
+            Row index is fh, with additional (upper) levels equal to instance levels,
+                from y seen in fit, if y_inner_mtype is Panel or Hierarchical.
+            Entries are forecasts of lower/upper interval end,
+                for var in col index, at nominal coverage in second col index,
+                lower/upper depending on third col index, for the row index.
+                Upper/lower interval end forecasts are equivalent to
+                quantile forecasts at alpha = 0.5 - c/2, 0.5 + c/2 for c in coverage.
+        """
+        model = self._fitted_forecaster
+        fh_int = fh.to_relative(self.cutoff)
+        steps = fh_int[-1]
+        n_lags = model.k_ar
+
+        y_cols_no_space = [str(col).replace(" ", "") for col in self._y.columns]
+
+        df_list = []
+
+        for cov in coverage:
+
+            alpha = -0.5 * cov + 0.5
```

Comment:
the docs say "significance level", so I think we should have `alpha = 1 - cov`, instead

Reply:
Corrected in the upcoming PR